django_project/purchases/api/filters.py

```diff
@@ -7,5 +7,6 @@ class PurchaseFilter(django_filters.FilterSet):
     class Meta:
         model = Purchase
         fields = {
-            'created_at': ['exact', 'lte', 'gte'],
+            'user__id': ['exact'],
```

Comment:
here is the problem? For exact rules could be only fields set
```suggestion
            fields = ('user__id', 'book__tilte')
```